media-gfx/solvespace/solvespace-2.3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+DXFRW_COMMIT="87ff1082f49b1286a033ac8f38fe8dde0975bb8b"
+DXFRW_PV="0.6.3"
+DXFRW_PN="libdxfrw"
+DXFRW_P="${DXFRW_PN}-${DXFRW_PV}-${DXFRW_COMMIT}"
+
+inherit cmake-utils
+
+DESCRIPTION="Parametric 2d/3d CAD"
+HOMEPAGE="http://solvespace.com"
+SRC_URI="https://github.com/solvespace/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Also please inline `${PN}` here, to make it easier to copy-paste the address to look at GitHub site.